src/modules/poweraccent/PowerAccent.Core/Services/SettingsService.cs

```diff
@@ -164,6 +164,9 @@ public static char[] GetDefaultLetterKey(LetterKey letter)
                 return new char[] { 'û', 'ù', 'ü', 'ú', 'ū' };
             case LetterKey.VK_Y:
                 return new char[] { 'ÿ', 'ý' };
+            case LetterKey.VK_MINUS:
+                // Left to right: Underscore(U+005F), figure dash (U+2012), en dash (U+2013), em dash (U+2014), horizontal bar (U+2015)
+                return new char[] { '_', '‒', '–', '—', '―' };
```

Comment:
is it correct that the last three looking the same?

Reply:
In the application only the last two look the same, but they aren't

Reply:
I'm slightly sceptical with the order though. Isn't the underscore easily reachable on the mostly used keyboard layouts? For me, being at the end of the list would make more sense.

Reply:
how does one know the difference between 2013 and 2014?

Reply:
2014 is (depending on the font) up to twice as wide as 2013.

Reply:
Tooltips would help alot. But that means a lot more text and it needs to be translated as wel.

Reply:
> Tooltips, or always visible subtext, would help alot. But that means a lot more text and it needs to be translated as wel.

When we would use just the official unicode name, we don't have to traslate it

Reply:
The subtext idea would be great since it would also enable the input of all the differently sized white space characters.

Reply:
dashes to me prove we need some subtext.  Visually i have no clue between 2012 to 2015.  Maybe at the bottom of the overlay, we have the code + name?

Reply:
> dashes to me prove we need some subtext.  Visually i have no clue between 2012 to 2015.  Maybe at the bottom of the overlay, we have the code + name?

@crutkas Yes, I think I will put this PR on blocked until we have a solution to do this.

Reply:
Also I want to up a technical issue. For French keyboard for example, `minus` key is the same as `6` key. I just made the test and `minus` key is completly not functional.
It's why I choose to put all `others` characters under `comma` key.

But I checked some keyboard layouts, and it seems it is concerning only for French layout. So let you decide about what you do :)